Project-II-Calculator/calculator/src/index.css

```diff
@@ -2,4 +2,8 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
```

Comment:
For your `index.css` file, you'll also want to add the following selector:
```
* {
  box-sizing: border-box;
}
```

It will prevent padding from adding extra height/width to an element (it's a good practice to include this behavior).